### PR TITLE
Add task manager

### PR DESCRIPTION
**Pull request by Dezmond-Huston**
Added the task manager to make and save tasks, and implemented check boxes to show which tasks have been completed.

### CODE REVIEW

File: mich_gecko.html [hunk 1]
```diff
@@ -6,6 +6,10 @@
     <title>Gecko 9 Team</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="gecko.css">
```
Comment (MichelleWasabi) [inline mich_gecko.html]: Hi Dez! I reviewed the code and found a bug on line 8. We need to modify the file name to match (mich_gecko.css)

**Review by MichelleWasabi — CHANGES_REQUESTED** [review]
Left a comment regarding changes in mich_gecko.css

File: mich_gecko.html (hunk 1 above)
Comment (Dezmond-Huston) [reply mich_gecko.html]: Huh. That's weird, I don't remember altering that. But that would explain why the background image had no styling, and didn't fit the page. I'm changing it now.

Comment (Dezmond-Huston) [reply mich_gecko.html]: I fixed it, and have pushed the changes. I'm closing this pull request, and opening one with the changes you suggested. If you don't find any bugs with this, could you merge it into the Develop Branch?

**Pull request closed without merging**